src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,14 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => (
+  <>
+    <div className="CommentList">
+      {
+      comments.map(comment => (
```

Comment:
```suggestion
  <div className="CommentList">
    {comments.map(comment => (
```
Fix formatting here too